### ISSUE TITLE
The generated image has poor quality

### ISSUE THREAD

**Issue by ffxxy** [LTH14/JiT#34]
Hi, I trained JIT-H/16 on my own data using the following commands:
CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 torchrun --nproc_per_node=8 --nnodes=1 --node_rank=0 main_jit.py \
    --model JiT-H/16 \
    --proj_dropout 0.2 \
    --P_mean -0.8 --P_std 0.8 \
    --noise_scale 1.0 \
    --batch_size 3 --blr 5e-5 \
    --epochs 600 --warmup_epochs 5 \
    --gen_bsz 16 --num_images 200 --interval_min 0.1 --interval_max 1.0 \
    --class_num 1 --label_drop_prob 0 --cfg 1.0 \
    --output_dir ${OUTPUT_DIR} --resume ${PRETRAIN_DIR} \
    --data_path ${IMAGENET_PATH} --online_eval

My data only has one category, human body. After training for 600 epochs, the generated data is very blurry and abstract. How can I optimize it?

<img width="281" height="533" alt="Image" src="https://github.com/user-attachments/assets/6dc0b02e-0437-4db3-9134-3d4a1b2df9d5" />

**Comment by yyhtbs-yye** [comment]
I have the similar issue. All human faces are really bad in quality. Only Animal category are OK some are nice. 

An image from H16 for 256x256. 
<img width="256" height="256" alt="Image" src="https://github.com/user-attachments/assets/08bb8287-609e-42c4-afba-fa3207ba5d36" />

**Comment by yankai233** [comment]
I used the JIT-B/32 provided by the author, but its quality is also very poor.

<img width="512" height="512" alt="Image" src="https://github.com/user-attachments/assets/1779d820-2f1b-4228-bc9b-49db21bc387f" />
<img width="512" height="512" alt="Image" src="https://github.com/user-attachments/assets/93bf8047-900a-4171-8862-7a01544e122a" />

**Comment by panjiashu** [comment]
Is FID metric consistent, though? This human face issue is pervalent in generative models trained only on ImageNet.

**Comment by yyhtbs-yye** [comment]
> Is FID metric consistent, though? This human face issue is pervalent in generative models trained only on ImageNet.

I think the FID is consistent for B/32 quite stable, did not tested H/32 for many rounds. Another problem is the background, in different round, wired periodic patterns show up in some images, not sure about the reasons (potentially the size of tha patch).
